test/src/test/java/hudson/model/ProjectTest.java

```diff
@@ -518,7 +520,11 @@ public void testGetRelationship() throws Exception {
         downstream.getBuildersList().add(new TestBuilder() {
             @Override public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) {
                 for (Run<?, ?>.Artifact a : upstream.getLastBuild().getArtifacts()) {
-                    Util.copyFile(a.getFile(), new File(build.getWorkspace().child(a.getFileName()).getRemote()));
+                    try {
+                        Files.copy(a.getFile().toPath(), new File(build.getWorkspace().child(a.getFileName()).getRemote()).toPath(), REPLACE_EXISTING);
+                    } catch (IOException e) {
+                        throw new RuntimeException(e);
```

Comment:
In situations like this I tend to prefer the more specific `UncheckedIOException`. In general this is considered good coding style; see e.g. [2. Prefer Specific Exceptions](https://stackify.com/best-practices-exceptions-java/).

Reply:
Thanks for the suggestion and also the link for clarification.